k4e-agent-build

```diff
@@ -0,0 +1,5 @@
+sudo GOPROXY=proxy.golang.org,direct PWD=$PWD spec=$PWD outdir=$PWD make -f .copr/Makefile srpm
+sudo rm -rf /root/rpmbuild/*
```

Comment:
Can this be reduced only for the specific folders/files created by the agent ?

Reply:
not at the moment. This is determined by Yggdrasil code and scripts. besides, it's just a manual procedure. Anyone can avoid this line or alter it.